http/data-url.json

```diff
@@ -0,0 +1,63 @@
+{
+  "version": "1.0.0",
+  "data": {
+    "http": {
+      "data-url": {
+        "__compat": {
+          "basic_support": {
+            "desc": "data URL scheme",
+            "support": {
+              "webview_android": {
+                "version_added": true
+              },
+              "chrome": {
+                "version_added": true
+              },
+              "chrome_android": {
+                "version_added": true
+              },
+              "edge": {
+                "version_added": "12",
+                "notes": ["IE9 and later, as well as Edge, supports data URLs in CSS and JS files, but not in HTML files, with a max size of 4GB."]
+              },
+              "edge_mobile": {
+                "version_added": true
+              },
+              "firefox": {
+                "version_added": true
+              },
+              "firefox_android": {
+                "version_added": true
+              },
+              "ie": {
+                "version_added": "8",
+                "notes": ["IE8 only supports data URLs in CSS files, with a max size of 32kB",
+                          "IE9 and later, as well as Edge, supports data URLs in CSS and JS files, but not in HTML files, with a max size of 4GB."]
```

Comment:
These two notes make me wonder if it's worth using subfeatures here for "CSS files"/"JS files", "HTML files".